src/schemas/json/jj.json

```diff
@@ -0,0 +1,136 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "Jujutsu VCS config",
+    "type": "object",
```

Comment:
If this top-level object specification can have a `description` field, I think it would make sense to make that say something like "See https://github.com/martinvonz/jj/blob/main/docs/config.md for details"

Reply:
Good idea! Added.